src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/cloud-watch-fallback.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "CloudWatch fallback"
+metaDescription: This doc guides you through shipping your telemetry data with Cloudwatch as a fail-safe option.
+freshnessValidatedDate: never
+---
+
+You can ship New Relic Lambda monitoring data in a few different ways depending on your needs. The option that provides the strongest performance of your function and the strongest fail-safe is the CloudWatch fallback option. This means that you will ship telemetry to New Relic through the agent and use the extension to enhance the telemetry collection but, if there's a problem with the extension, you will still receive logs and payloads in CloudWatch.  
+
+While this offers the strongest fail-safe, it's important to note that the AWS CloudWatch service can generate a lot of data. Keep data ingest in mind when thinking about costs and which New Relic pricing plan you choose. 
+
```

Comment:
How CloudWatch fallback works:

If the Extension fails to start up (noop mode) or crashes, it will fail to encode the `NR_LAMBDA_MONITORING` line generated by our agents, thus the `NR_LAMBDA_MONITORING` line will appear in CloudWatch logs. If the [aws-log-ingestion](https://github.com/newrelic/aws-log-ingestion) function is installed in the same region as your function, configured to ship payloads, and is triggered by your function's CloudWatch log group, then it can act as a fallback to the extension for shipping payloads.

Reply:
How about this:

For each scenario, if the Extension fails to start up, noop mode, or crashes, it will fail to encode the `NR_LAMBDA_MONITORING` line generated by our agents. The `NR_LAMBDA_MONITORING` line will appear in CloudWatch logs instead. If the `aws-log-ingestion` function is installed in the same region as your function, configured to ship payloads, and is triggered by your function's CloudWatch log group, then it can act as a fallback to the extension for shipping payloads.

Reply:
This doesn't apply to each scenario, only the first two. The third has the extension disabled completely.

Reply:
I fixed this!

Reply:
I don't know if I'd agree that logging to CloudWatch is a fallback as such. It's an alternative delivery mechanism for telemetry and logs. It *can be* a fallback, but I'd say that's a question for the use case the customer has in mind. 

Would it be possible to frame it in a more neutral way, and possibly explain why we do this the way we do? As an example, something like this:

In normal APM mode, New Relic agents connect to New Relic for configuration data, and then send telemetry and logs at regular intervals. Serverless mode is different, because serverless invocations often have very short durations--a few seconds, or under a second. So when they operate in serverless mode, our agents rely on some other process to send telemetry in logs. The alternatives are
* The New Relic Lambda Extension (brief explanation)
* The newrelic-log-ingestion Lambda function, triggered by a CloudWatch Log Subscription (brief explanation)